native/cocos/core/scene-graph/Node.h

```diff
@@ -655,6 +663,10 @@ class Node : public BaseNode {
              _localScale.x, _localScale.y, _localScale.z);
     }
 
+    inline bool hasChanged() const {
```

Comment:
Please use more meaningful function name.

Reply:
I have removed this method, it is only used once